client/src/components/Search/Search.js

```diff
@@ -38,88 +38,88 @@ function hasFilters(filters) {
  * Displays a search component that can be use to retrieve a search term and
  * display an advanced search form.
  */
-class Search extends Component {
-  constructor(props) {
-    super(props);
-
-    this.expand = this.expand.bind(this);
-    this.handleChange = this.handleChange.bind(this);
-    this.getData = this.getData.bind(this);
-    this.doSearch = this.doSearch.bind(this);
-    this.focusInput = this.focusInput.bind(this);
-    this.focusFirstFormField = this.focusFirstFormField.bind(this);
-    this.hide = this.hide.bind(this);
-    this.show = this.show.bind(this);
-    this.toggle = this.toggle.bind(this);
-    this.open = this.open.bind(this);
-    this.searchTermIsDirty = this.searchTermIsDirty.bind(this);
-    this.clearFilters = this.clearFilters.bind(this);
-    this.clearSearchBox = this.clearSearchBox.bind(this);
-    this.clearFormFilter = this.clearFormFilter.bind(this);
-    this.focusFormFilter = this.focusFormFilter.bind(this);
-    this.formatTagData = this.formatTagData.bind(this);
-    this.onClickOut = this.onClickOut.bind(this);
-
-    const term = props.term
-      || (props.filters && props.filters[`${props.filterPrefix}${props.name}`])
-      || '';
-
-    this.state = {
-      display: props.display,
-      searchText: term,
-      initialSearchText: term,
-      forceLoadForm: false,
-    };
-  }
-
-  componentDidMount() {
-    this.setOverrides(this.props);
-  }
-
-  componentWillUnmount() {
-    this.setOverrides();
-  }
+const Search = (props) => {
+  const {
```

Comment:
The following props were removed - defined in propTypes but not used in code:
- tagHandlers,
- borders,

They are still passed into `<SearchBox>`